drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1515,8 +1522,24 @@ function DragAndDropBlock(runtime, element, configuration) {
             };
 
             var raf_id = null;
+            var last_target_zone = null;
             var onDragMove = function(evt) {
                 evt.preventDefault();
+                var $zone = getTargetZone(evt);
```

Comment:
is the naming of the variable intentional? Else it creates a confusion with jQuery syntax

Reply:
I try to keep consistency on names, on other part of the code has the same name also it responds as a JQuery type variable

Reply:
Done